examples/headless/src/main.rs

```diff
@@ -92,15 +92,15 @@ async fn render(mut scenes: SceneSet, index: usize, args: &Args) -> Result<()> {
     let mut builder = SceneBuilder::for_fragment(&mut fragment);
     let example_scene = &mut scenes.scenes[index];
     let mut text = SimpleText::new();
-    let mut params = SceneParams {
+    let mut scene_params = SceneParams {
         time: args.time.unwrap_or(0.),
         text: &mut text,
         resolution: None,
```

Comment:
This doesn't compile. That is, `headless` also needs to handle the case where this is set in the scene.

Hopefully we can get #250 sorted soon.

Reply:
Oops! Done.